compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -1335,6 +1361,30 @@ void propagate_concat_quantparam(luci::CircleConcatenation *concat, loco::DataTy
  */
```

Comment:
I this it would be better to update this comment as well. For example,

 `NOTE: qparam is propagated from input to output if padded value is the lowest float value`